deploy-cluster-aws/fabfile.py

```diff
@@ -138,13 +133,26 @@ def configure_bigchaindb():
     run('bigchaindb -y configure', pty=False)
 
 
+# Send the specified configuration file to
+# the remote host and save it there in
+# ~/.bigchaindb
+# Use in conjunction with set_host()
+# No @parallel
+@task
+def send_confile(confile):
+    put('confiles/' + confile, 'tempfile')
+    sudo('mv tempfile ~/.bigchaindb')
```

Comment:
do we need sudo here?
I am afraid that the bigchaindb process may not have the right read/write permissions